physionet-django/physionet/models.py

```diff
@@ -4,14 +4,23 @@
 
 
 class StaticPage(models.Model):
+    """
+    Allows pages on the site to be created via the admin tool. Controls whether a given page appears in the navigation
+    bar.
+    """
     title = models.CharField(max_length=64)
     url = models.CharField(max_length=64, unique=True)
+    nav_bar = models.BooleanField(default=False)
+    nav_order = models.IntegerField(unique=True, null=True)
```

Comment:
1. Since this column defines the order, maybe it makes more sense to support only positive integers?
2. Setting just `null=True` will only prevent the db from setting the `NOT NULL` constraint but all the Django forms and logic will still treat this field as required - e.g. now it's impossible to create a new static page that does not have this field set through the admin console. Adding `blank=True` allows this field to not be set through a form (I assume that was the expectation)
```suggestion
    nav_order = models.PositiveSmallIntegerField(unique=True, null=True, blank=True)
```

Reply:
Thanks @kshalot , these are good suggestions! I've implemented them.